site/assets/css/inline.css

```
@@ -138,6 +138,8 @@ body {
   border-right: 1px solid rgba(0, 0, 0, 0.1);
   background-color: #fafafa;
   z-index: 99999;
+  overflow-y: scroll;
```

this should be:
```suggestion
  overflow-y: auto;
```
we don't want to show a scrollbar when it's not needed